python/idsse_common/test/test_grid_proj.py

```diff
@@ -33,13 +38,13 @@
 ]
 EXAMPLE_PROJECTIONS = [
     (-126.2766, 19.229),
-    (-126.27660549554, 19.229022224607),
-    (-126.23803580508, 19.272773488997)
+    (-126.28209649530142, 19.251224896946418),
+    (-71.12860760432866, 54.09108720984939)
 ]
 EXAMPLE_GEOS = [
-    (-3275807.350733, -260554.63043505),
-    (-3275807.350733, -260552.09073205),
-    (-3270727.944733, -256745.07593505)
+    (-3275807.350733357, -260554.63043505285),
+    (-3275807.3507333556, -258014.92743505232),
+    (1803598.6492666428, 3548999.8695649463)
```

Comment:
Good cleanup, probably best to get the actual expected values than have to round everything to do assert comparisons